Letterbook.Core/Models/TimelineEntry.cs

```diff
@@ -6,11 +6,23 @@ public class TimelineEntry
     // Those are often more or less the same, except in the case of boosts
     public DateTime Time { get; set; } = DateTime.UtcNow;  //index
     public ActivityObjectType Type { get; set; }
-    public string EntityId { get; set; }  //index
-    public string AudienceKey { get; set; } //index
+    public required string EntityId { get; set; }  //index
```

Comment:
I didn't know this was a thing, TIL!

Reply:
I think it's a relatively recent thing. It might have been introduced in dotnet 6?